cmd/cluster/support/post.go

```diff
@@ -1,319 +1,203 @@
 package support
 
 import (
-	"encoding/json"
+	"bytes"
 	"fmt"
-	"log"
-	"net/http"
-	"net/url"
-	"os"
-	"strings"
-
-	"github.com/openshift-online/ocm-cli/pkg/arguments"
 	"github.com/openshift-online/ocm-cli/pkg/dump"
 	sdk "github.com/openshift-online/ocm-sdk-go"
-	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
-	"github.com/openshift/osdctl/internal/support"
-	"github.com/openshift/osdctl/internal/utils"
-	"github.com/openshift/osdctl/internal/utils/globalflags"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	slv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
 	ctlutil "github.com/openshift/osdctl/pkg/utils"
 	"github.com/spf13/cobra"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
-	cmdutil "k8s.io/kubectl/pkg/cmd/util"
-)
-
-var (
-	template                                                string
-	templateParams, userParameterNames, userParameterValues []string
+	"os"
 )
 
 const (
-	defaultTemplate = ""
+	LimitedSupportSummary         = "Cluster is in Limited Support due to unsupported cluster configuration"
+	ProblemFlag                   = "problem"
+	ResolutionFlag                = "resolution"
+	EvidenceFlag                  = "evidence"
+	InternalServiceLogSeverity    = "Warning"
+	InternalServiceLogServiceName = "SREManualAction"
+	InternalServiceLogSummary     = "LimitedSupportEvidence"
 )
 
-type postOptions struct {
-	output         string
-	verbose        bool
-	clusterID      string
-	isDryRun       bool
-	limitedSupport support.LimitedSupport
+func newCmdpost() *cobra.Command {
 
-	genericclioptions.IOStreams
-	GlobalOptions *globalflags.GlobalOptions
-}
-
-func newCmdpost(streams genericclioptions.IOStreams, globalOpts *globalflags.GlobalOptions) *cobra.Command {
-
-	ops := newPostOptions(streams, globalOpts)
 	postCmd := &cobra.Command{
-		Use:               "post CLUSTER_ID",
-		Short:             "Send limited support reason to a given cluster",
+		Use:   "post CLUSTER_ID",
+		Short: "Send limited support reason to a given cluster",
+		Long: `Sends limited support reason to a given cluster, along with an internal service log detailing why the cluster was placed into limited support.
+The caller will be prompted to continue before sending the limited support reason.`,
+		Example: `# Post a limited support reason for a cluster
+osdctl cluster support post 1a2B3c4DefghIjkLMNOpQrSTUV5 --problem="the cluster has a second failing ingress controller, which is not supported and can cause issues with SLA" \
+--resolution="remove the additional ingress controller 'my-custom-ingresscontroller'. 'oc get ingresscontroller -n openshift-ingress-operator' should yield only 'default'" \
+--evidence="See OHSS-1234"`,
```

Comment:
Should we put more in the evidence example? I'm just concerned people will think they _need_ to make an OHSS when they could just provide something like. "I went to <splunk link> and saw modifications to etcd by <user> on <datetime>."